e2e/src/fula.test.ts

```diff
@@ -0,0 +1,151 @@
+import test from 'tape';
+import {createClient, Status} from '@functionland/fula';
+import {FileTest, GraphQL, GraphQLSubscription} from "./test-data";
+import wrtc from "wrtc"
+import * as process from "process";
+
+let TIMEOUT
+let serverIds
+
+const beforeRun = ()=>{
+  if(process.env.SERVER_ID){
+    serverIds = process.env.SERVER_ID
```

Comment:
ahh gotcha - should we rename to BOX_ID (for consistency) and include in README?

Reply:
Done